src/workloads/execution/CreateIndex.yml

```diff
@@ -0,0 +1,163 @@
+# This workload inserts documents in the first phase containing all types of indexes. The second phase creates indexes for each of them.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 150000
+    BatchSize: 10000
+    Document:
+      a: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
+      b: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}]
+      c: { type: "Point", coordinates: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}] }
+      d: {^RandomString: {length: 50}}
+      e: {^RandomInt: {min: -10000000, max: 10000000}}
+      "attributes": { "color": "red", "size":" large", "inside": { "bookmark": 1, "postitnote":2 }, "outside": { "dustcover": "worn" }}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+  - &Nop {Nop: true}
+
+- Name: IndexCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: Create2dIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            a: "2d"
+            min: -180
+            max: 180
+            bits: 32
+          name: 2dIndex
+    - OperationMetricsName: DropCreated2dIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: 2dIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: Create2dSphereIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key:
+            b: "2dsphere"
+            c: "2dsphere"
+            name: 1
+          name: 2dsphereIndex
+    - OperationMetricsName: DropCreated2dSphereIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        dropIndexes: Collection0
+        index: 2dsphereIndex
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: CreateGeoHaystackIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            a: "geoHaystack"
+            d: 1
```

Comment:
Remove the compound index key here

Reply:
For GeoHaystack indexes, I need to provide an additional key. Otherwise I get this error:

`[genny::ServerResponse*] = { "ok" : 0.0, "errmsg" : "no non-geo fields specified", "code" : 16774, "codeName" : "Location16774" }`

Reply:
Acknowledge